src/sentry/dynamic_sampling/rules/biases/boost_latest_releases_bias.py

```diff
@@ -69,15 +70,12 @@ def _generate_bias_rules(self, bias_data: BiasData) -> List[PolymorphicRule]:
                     },
                     "id": bias_data["id"] + idx,
                     "timeRange": {
-                        "start": str(
-                            datetime.utcfromtimestamp(boosted_release.timestamp).replace(tzinfo=UTC)
-                        ),
-                        "end": str(
-                            datetime.utcfromtimestamp(
-                                boosted_release.timestamp
-                                + boosted_release.platform.time_to_adoption
-                            ).replace(tzinfo=UTC)
+                        "start": datetime.utcfromtimestamp(boosted_release.timestamp).strftime(
+                            self.date_format
                         ),
+                        "end": datetime.utcfromtimestamp(
+                            boosted_release.timestamp + boosted_release.platform.time_to_adoption
+                        ).strftime(self.date_format),
```

Comment:
This change is not necessary functionally, but it makes sure that Relay's serialized project config look the same as Sentry's serialized project config.